core/tests/performance_framework/perf_domain.py

```diff
@@ -236,8 +236,8 @@ def print_details(self):
         """Helper function to print details for all the events."""
         if self.page_session_stats:
             print 'Total number of requests: %d' % self.get_request_count()
-            print ('Total page size in bytes: %d'
-                   % self.get_total_page_size_bytes())
+            print('Total page size in bytes: %d'
```

Comment:
This will unfortunately change our print style (since we don't use `print(...)` convention). Since the Google Style [guide](https://google.github.io/styleguide/pyguide.html?showone=Strings#Strings) on strings do allow the old method, we should follow it. Will it be possible to make an exception for this type of rule?

Reply:
This goes for the same elsewhere.

Reply:
Reverted this at all places. Thanks!